src/views/Auth.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { signInUser, signupUser } from '../services/users';
+import AuthForm from '../components/AuthForm';
+import classNames from 'classnames';
+
+export default function Auth({ setCurrentUser }) {
+  const [type, setType] = useState('signin');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    try {
+      const resp =
+        type === 'signin' ? await signInUser(email, password) : await signupUser(email, password);
+      setCurrentUser(resp);
+    } catch {
+      setErrorMessage('Something went wrong. Please try again');
+    }
+  };
+
+  return (
+    <div>
+      <div>
+        <h2
+          onClick={() => {
+            setType('sign in');
+          }}
+          className={classNames({ active: type === 'sign in' })}
```

Comment:
You will some CSS targeting the `active` selector for the classnames library to do anything